helper/clientInitialize.go

```diff
@@ -24,30 +24,86 @@ func getAppknoxAccessToken() string {
 	return accessToken
 }
 
+// GetHostMappings returns a map of host names to URLs.
+ func GetHostMappings() map[string]string {
+     return map[string]string{
+         "global": "https://api.appknox.com/",
+         "saudi":   "https://sa.secure.appknox.com/",
+         // Add more mappings as needed
+     }
+ }
+
+func getAPIHostMappings() map[string]string {
+    // Instead of using an environment variable, call the new function
+    return GetHostMappings()
+}
+
```

Comment:
Is the function needed? `GetHostMappings` doing the same thing.

Reply:
Yes, the function is redundant, fixed that.